discovery-provider/src/tasks/index_solana_plays.py

```diff
@@ -313,7 +334,7 @@ def parse_sol_tx_batch(
     challenge_bus = index_solana_plays.challenge_event_bus
 
     # Process each batch in parallel
-    with concurrent.futures.ThreadPoolExecutor(max_workers=5) as executor:
+    with concurrent.futures.ThreadPoolExecutor(max_workers=10) as executor:
```

Comment:
fyi

default
ThreadPoolExecutor: min(32, (os.cpu_count() or 1) + 4)

https://stackoverflow.com/questions/66798981/working-of-concurrent-futures-threadpoolexecutor-max-workers-when-scaling-up-the

probably we should leave in the default so it is programmatic with num CPUs